### PR TITLE
Hotfix: Freemius update

### PR DESCRIPTION
**Pull request by Camwyn**
Not sure if we care about the mode changes, the rest looks legit.

[TEC-4430]

[TEC-4430]: https://theeventscalendar.atlassian.net/browse/TEC-4430?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Camwyn** [comment]
Both allow and skip appear to work as expected on new installs of TEC or ET.
The **only** issue I have seen is dismissing the notices when both are installed - one of them needs to be dismissed twice.

**Comment by borkweb** [comment]
It looks like a submodule hash update for tribe-common-styles snuck into this PR

**Comment by Camwyn** [comment]
> It looks like a submodule hash update for tribe-common-styles snuck into this PR

Fixed - thanks!

**Review by EugeneKyale — CHANGES_REQUESTED** [review]
Can we remove all these empty files?

**Review by borkweb — APPROVED** [review]

**Review by EugeneKyale — APPROVED** [review]

File: vendor/freemius/includes/class-freemius.php [hunk 1]
```diff
@@ -4058,7 +4058,7 @@ function get_anonymous_id( $blog_id = null ) {
             if ( empty( $unique_id ) || ! is_string( $unique_id ) ) {
                 $key = fs_strip_url_protocol( get_site_url( $blog_id ) );
 
-                $secure_auth = SECURE_AUTH_KEY;
+                $secure_auth = defined( 'SECURE_AUTH_KEY' ) ? SECURE_AUTH_KEY : '';
```
[the remark this answers is not in the document]
Comment (borkweb) [reply vendor/freemius/includes/class-freemius.php:4061]: This is code from the Freemius SDK, so we should not modify it.

[the remark this answers is not in the document]
Comment (Camwyn) [reply vendor/freemius/includes/class-freemius.php:4061]: This isn't our code - we're not going to edit it

**Pull request merged**